scene/2d/audio_stream_player_2d.cpp

```diff
@@ -448,6 +471,29 @@ Ref<AudioStreamPlayback> AudioStreamPlayer2D::get_stream_playback() {
 	return stream_playback;
 }
 
+void AudioStreamPlayer2D::set_distance_drawing_enabled(bool enable) {
```

Comment:
```suggestion
void AudioStreamPlayer2D::set_distance_drawing_enabled(bool p_enable) {
```
(Same for the other one and in the header.)